starter-core/src/main/java/io/micronaut/starter/feature/build/MicronautAot.java

```diff
@@ -65,17 +71,28 @@ public boolean supports(ApplicationType applicationType) {
         return applicationType == ApplicationType.DEFAULT;
     }
 
+    @Override
+    public boolean isPreview() {
+        return true;
```

Comment:
This isn't a preview anymore since we released 1.0.0.

Reply:
I've added a comment in the build files instead.